automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookPyKernelSpec.scala

```diff
@@ -58,4 +67,80 @@ class NotebookPyKernelSpec extends RuntimeFixtureSpec with NotebookTestUtils {
     }
 
   }
+
+  "RuntimeSystemSpec" - {
```

Comment:
moved this spec here since it can all be done on the same runtime, to reduce parallelization